src/2ls/preprocessing_util.cpp

```diff
@@ -849,3 +849,43 @@ void twols_parse_optionst::make_symbolic_array_indices(goto_modelt &goto_model)
   }
   goto_model.goto_functions.update();
 }
+
+/// Makes user input nondeterministic, i.e. arguments of fscanf starting
+/// from the second one are assigned a nondeterministic value.
+void twols_parse_optionst::make_scanf_nondet(goto_modelt &goto_model)
```

Comment:
IIRC I saw Michael fixing this in CBMC so it should already be fixed, we just need to update again, so this is a temporary workaround :)

Reply:
I thought about cherry-picking that fix to our fork, but I don't think it would have been that straight forward, there were more changes in the source code of C function models.

Reply:
Ok, then we can revert on the next CBMC prerequisites upgrade.